CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Breaking changes
+
+- The anchoring chain observer logic has been moved to the `before_commit` stage. (#131)
+  Thus additional thread in the public api handler has been no longer used.
```

Comment:
Is it really important for user?

Reply:
In theory this is a deeply internal change.